app/ProjectAanvraag/Listeners/CreateWidget.php

```diff
@@ -139,4 +140,24 @@ private function handle(UuidInterface $integrationId): void
             )
         );
     }
+
+    public function failed(IntegrationCreated|ContactCreated|ConsumerCreated $event, Throwable $throwable): void
+    {
+        $entity = match (get_class($event)) {
+            IntegrationCreated::class => 'integration',
+            ContactCreated::class => 'contact',
+            ConsumerCreated::class => 'consumer',
+        };
+
+        $id = match (get_class($event)) {
+            IntegrationCreated::class => $event->integrationId->toString(),
```

Comment:
Feel free to change `integrationId` to `id`

Reply:
[Rename integrationId to id](https://github.com/cultuurnet/publiq-platform/pull/906/commits/3a37ec41b0470cc205ae6f9f36420399cacda992)